x-pack/test/alerting_api_integration/security_and_spaces/tests/actions/index.ts

```diff
@@ -21,5 +21,6 @@ export default function actionsTests({ loadTestFile }: FtrProviderContext) {
     loadTestFile(require.resolve('./builtin_action_types/email'));
     loadTestFile(require.resolve('./builtin_action_types/es_index'));
     loadTestFile(require.resolve('./builtin_action_types/pagerduty'));
+    loadTestFile(require.resolve('./builtin_action_types/webhook'));
```

Comment:
I think we can add some FT to test the whitelisting; the config we're using only has `localhost`, so you should fail trying to create a webhook pointing anywhere else.  Seems hard at the moment to test whitelist failure during execute, since you'd have to create the webhook action first, which would mean restarting Kibana with a different whitelist config in between, or some low-level hacking of the action saved object, which will be may be even more complicated since it's an encrypted saved object.

Reply:
Added a missing test for creation stage and added a card in the project for the inability to test executor stage.

https://github.com/elastic/kibana/projects/26#card-26053651